customization/parallel-data.md

```diff
@@ -0,0 +1,31 @@
+---
+description: Parallel data for machine translation
+---
+
+# Parallel data
+
+**Parallel data** or **parallel corpora** are datasets of **translation pairs** - sentences and their translations. They are used to train and test machine translation models.
+
+| Original | Translation |
+| ----------- | ----------- |
+| File | Archivo |
+
+Parallel datasets can include translations for one or more language pairs, and be directioned or directionless.
+
+Parallel data can be created by crawling and aligned monolingual test, and by [back-translation](customization/back-translation.md) or [back-copying](customization/back-translation.md).
+
+### Goals
+
+Parallel data is necessary to train statistical and neural machine translation engines.
+
+### Challenges
+
+Parallel data is available in most widely spoken languages, but not available in other language pairs.
```

Comment:
widely *written*, but even this is problematic, because there is not much parallel data for 2 big languages that don't have a lot of translation need.  (e.g. EN:PL >> HI:ES)

Also, parallel data is not *in* a language, it's *for* a language *pair*.